### DISCUSSION TITLE
responsiveness

### DISCUSSION THREAD

**Discussion by Rolexero** [saadeghi/daisyui#943] [General]
first time trying daisy UI and it's not responsive

**Comment by AshSimmonds** [comment]
Wonder if it's similar issue I'm having on #908 - haven't had a chance to investigate for a couple weeks.

Can you try adding this code in your page somewhere - all it does is highlight what media query (page size) it's seeing. Sometime in the last month mine is only seeing `mobile`, no matter if page is resized to `sm` or `md` or `xl`.

```
<div class="m-auto items-center "> 
    <span class="tooltip tooltip-bottom badge bg-primary text-xs" data-tip="mobile">PH</span>
    <span class="tooltip tooltip-bottom badge sm:bg-primary text-xs" data-tip="640px">SM</span> 
    <span class="tooltip tooltip-bottom badge md:bg-primary text-xs" data-tip="768px">MD</span> 
    <span class="tooltip tooltip-bottom badge lg:bg-primary text-xs" data-tip="1024px">LG</span> 
    <span class="tooltip tooltip-bottom badge xl:bg-primary text-xs" data-tip="1280px">XL</span> 
    <span class="tooltip tooltip-bottom badge 2xl:bg-primary text-xs" data-tip="1536px">2XL</span> 
</div>
```

**Comment by saadeghi** [comment]
You should be able to use any Tailwind CSS responsive class names and you should also be able to use responsive prefixes (`sm:`, `md:`, `lg:`, etc.) on any daisyUI class names that says `responsive` in the component class table ([example](https://daisyui.com/components/button/))

If it's not working, something is wrong about your config or build. If you provide a sample repo, we can help.

**Comment by Rolexero** [comment]
Ooh alright then, would check this
